ssl/statem/statem_clnt.c

```diff
@@ -1901,6 +1902,13 @@ MSG_PROCESS_RETURN tls_process_server_certificate(SSL *s, PACKET *pkt)
         x = NULL;
     }
 
+    /*
+     * Prior to TLS 1.3 the OCSP responses are sent in the CERT_STATUS message
+     * after the SERVER_CERTIFICATE message. Therefore the verification of the
+     * OCSP response here works only for TLS 1.3.
+     * Prior to TLS 1.3 the verification needs to be done in the client
```

Comment:
As others have pointed out, consider making this just a 1.3 feature; it simplifies things.

Reply:
This patch includes the multistapling only for TLS 1.3. I actually have an patch for TLS 1.2 as well, but because the OCSP responses in TLS 1.2 are sent at the end of the TLS handshake, the verify code would not have worked or only with major changes.
And because OCSP multistapling in TLS 1.2 (RFC 6961) is deprecated, I decided to only add the TLS 1.3 code.